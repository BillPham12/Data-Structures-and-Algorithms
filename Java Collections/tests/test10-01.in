1234
12
1234567
123456789

12345678
1
12345678
1234
123
123456
123
