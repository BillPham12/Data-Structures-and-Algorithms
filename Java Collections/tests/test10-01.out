123456789
12345678
12345678
123456
123
